Makefile

```diff
@@ -15,6 +15,11 @@ INCLUDES=
 LIBS=		-lm -lz -lpthread
 SUBDIRS=	.
 
+ifneq ($(simde),)
+	CFLAGS += -DUSE_SIMDE -DSIMDE_ENABLE_NATIVE_ALIASES -DSIMDE_ENABLE_OPENMP -fopenmp-simd
```

Comment:
Here is the explanation of `-fopenmp-simd`.
https://gcc.gnu.org/legacy-ml/gcc-patches/2013-10/msg02275.html

And here is the explanation of `SIMDE_ENABLE_OPENMP` in the simde `README.md`.
> For best performance, in addition to -O3 (or whatever your compiler's equivalent is), you should enable OpenMP 4 SIMD support by defining SIMDE_ENABLE_OPENMP before including any SIMDe headers, and enabling OpenMP support in your compiler.